poly/src/evaluations/multivariate/multilinear/mod.rs

```diff
@@ -57,7 +57,6 @@ pub trait MultilinearExtension<F: Field>:
     + CanonicalSerialize
     + CanonicalDeserialize
     + for<'a> AddAssign<&'a Self>
-    + for<'a> AddAssign<(F, &'a Self)>
```

Comment:
Why was this bound removed?

Reply:
I think this was seen as a false positive with `for<'a> AddAssign<&'a Self>`, just fixed